Instrukcije_d001/src/app/terminal.c

```diff
@@ -0,0 +1,131 @@
+//
+// Created by Edin on 15. 03. 2022.
+//
+
+#include "terminal.h"
+
+int stevec = 0; // counter za to kdo bo na vrsti (X ali 0)
```

Comment:
Te spremenljivke lahko daš v strukturo `Terminal` pa jaz bi jih mogoče poimenoval bolj splošno da poveš kako se te spremenljivke nanašajo na terminal... recimo...

* stevec => frames (frames je v game dev. industriji koliko krat je display prikazal frame (FPS frames per seconds))
* igreJeKonec => active (ali je terminal aktiven ali je mrtu) - To spremenljivko bi dodal v svet saj svet ve kdaj ga je konec xD terminal pa bo naredil kar more ko bo videl da se je svet šel v apokalipso xD